mkdocs.yml

```diff
@@ -143,7 +143,7 @@ pages:
         - 'API search result pagination': 'cookbook/paginating_api_search_results.md'
         - 'AWS S3 clustering': 'cookbook/setting_up_amazon_aws_s3_clustering.md'
         - 'Binary attachments via REST API': 'cookbook/creating_content_with_binary_attachments_via_rest_api.md'
-        - 'Creating block with `richtext` attribute type': 'cookbook/richtext_block.md'
+        - 'Creating block with richtext attribute type': 'cookbook/richtext_block.md'
```

Comment:
The title is still too long for a TOC

Reply:
I know, I haven't pushed the changes yet :D